docs/Release Notes/release-notes.md

```diff
@@ -6,7 +6,88 @@ slug: /release-notes/2022
 
 ---
 
-## v0.1.4
+## v0.1.9
+
+**November 25, 2022**
+
+**paralus/paralus**
```

Comment:
this has helm-changes as well.. let's capture that too..

https://github.com/paralus/helm-charts/pull/67

Reply:
Fixed.